x/hold/keeper/invariants.go

```diff
@@ -44,7 +45,7 @@ func holdAccountBalancesInvariantHelper(ctx sdk.Context, keeper Keeper) (string,
 	// So, long story short (too late), if the block time is zero, also bypass the vesting locked funds getter.
 	blockTime := ctx.BlockTime()
 	if blockTime.IsZero() {
-		ctx = ctx // banktypes.WithVestingLockedBypass(ctx) // TODO[1760]: locked-coins
+		ctx = banktypes.WithVestingLockedBypass(ctx)
```

Comment:
The `WithVestingLockedBypass` is only needed in this limited situation. It exists because the inclusion of the vesting locked-coins info is pretty deep in some 3rd-party library code. This is the only way to do that.

Further, the `WithVestingLockedBypass` follows a pattern used by other injectable functions to allow bypassing specific portions of those injected functions.